### DISCUSSION TITLE
Latest date as default value in variable drop-down

### DISCUSSION THREAD

**Discussion by psxpa3** [grafana/grafana#57569] [Dashboards, Panels & Visualizations]
Hi, I have a variable with dates in this format - 10-15-2022, 11-01-2022, 11-15-2022..etc.
On selecting Numeric (desc), the drop-down displays dates in decreasing order, however the lowest date is being selected as default. I would like the latest date to be as default value. I could not find any option in the variable creation etc. Please provide direction. Thank you

**Comment by tedraymond** [comment]
This can be done by sorting the variable, desc to have the latest date.  I accomplished this this week.  Then in the json scroll to the bottom to find your variable and there will be 2 lines, “text”=“your date”
“Selected”=“your date”

set both to =“”, then on dashbpoard load or the next day the top of the list variable will be default.

i have daily tables that populate in the same manner,  table_name_2022_10_26 for example.

If you want more info let me know

**Reply by psxpa3** [reply]
@tedraymond  thank you for your reply.
I can see this in my JSON Model of grafana
""
{
        "allValue": "auto",
        "current": {
          "selected": false,
          "text": "10-24-2022",
          "value": "10-24-2022"
        },
""

so, it seems I need to empty "text", and "value" here. Since the variable is already marked as Numeric(desc), hence this should be re-loaded with the latest date. Right?. Thank you very much for your help.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
